import-service/src/import-products-file.js

```diff
@@ -0,0 +1,35 @@
+import * as AWS from 'aws-sdk';
+import allowedOrigins from './constants/alloweb-origins';
+const { IMPORT_BUCKET_NAME, REGION } = process.env;
+
+export const importProductsFile = async (event) => {
+  console.log('Lambda invocation with event: ', event);
+  const catalogName = event.queryStringParameters.name;
+  const catalogPath = `uploaded/${catalogName}`;
+  const response = {
+    statusCode: 200,
+  };
+
+  const s3 = new AWS.S3({ region: REGION });
+  const params = {
+    Bucket: IMPORT_BUCKET_NAME,
+    Key: catalogPath,
+    Expires: 60,
+    ContentType: 'text/csv',
+  };
+
+  try {
+    const origin = event.headers.origin;
+    if (allowedOrigins.includes(origin)) {
```

Comment:
I was just wandering, as you already have cors restriction inside .serverless file, does it mean that `importProductsFile` lambda will not be executed if origin is anything except "https://dm9otfstrcg58.cloudfront.net"? If so, origin check is redundant here.

https://github.com/ihargit/nodejs-aws-be/pull/4/files#diff-fbe2ef5e7c375cfca379c36e1527e3e46a82742ae33b5527ad0124a7bae787b6R49-R58

Reply:
Hi, possible yes. But here it can be easier expanded to several origins. So, generally, it's ok to left check here, I think.

Reply:
But if you would like to add another origin you should modify .serverless file too

Reply:
It's true. Is it neccessary to delete it from serverless regarding the scope of the task?

Reply:
No, it's great, i was just interested)
Well done!